tools/api-markdown-documenter/src/documentation-domain-to-html/test/HeadingToHtml.test.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { h } from "hastscript";
+
+import { HeadingNode } from "../../documentation-domain/index.js";
+import { assertTransformation } from "./Utilities.js";
+
+describe("HeadingNode -> Html", () => {
+	it("With ID", () => {
+		const input = HeadingNode.createFromPlainText("Foo", "foo-id");
+		const expected = h("h1", { id: "foo-id" }, "Foo");
+		assertTransformation(input, expected);
+	});
+
+	it("Without ID", () => {
+		const input = HeadingNode.createFromPlainText("Foo", "foo-id");
+		const expected = h("h1", { id: "foo-id" }, "Foo");
+		assertTransformation(input, expected);
+	});
```

Comment:
Seems like the second one needs to be updated.

Reply:
🤦‍♂️